docs/Getting Started/Adding the Lock Screen/index.md

```diff
@@ -98,7 +98,19 @@ lockScreenConfig.setAppIcon(appIconInt); // For example, R.drawable.lockscreen i
 
 ### Showing the OEM Logo
 @![iOS]
-The default lock screen handles retrieving and setting the OEM logo from head units that support this feature. Currently this feature can not be disabled on the default lock screen.
+The default lock screen handles retrieving and setting the OEM logo from head units that support this feature. This feature can disabled on the default lock screen by setting `showDeviceLogo` to false. 
```

Comment:
Since Android and iOS now have the same logic for showing the OEM logo, the intro paragraph should be used for both platforms.

Reply:
fixed

Reply:
Now the Android platform has 2 intro paragraphs. You should use one or the other or combine them.